### DISCUSSION TITLE
Could not find mapping for model (but view is defined)

### DISCUSSION THREAD

**Discussion by flaviovich** [prisma/prisma#18230] [Q&A] [answered]
Hi!
I'm having troubles using views.
I'm getting this error message:
> Could not find mapping for model

But I'm a view was defined, this it is:
```
view resultado_evaluacion {
    IdColaborador Int     @unique @default(0) @db.UnsignedMediumInt
    Sede          String? @db.VarChar(45)
    Colaborador   String? @db.VarChar(45)
    Nivel         Int     @default(0)
    Categoria     String? @db.VarChar(45)
}
```
My generator:
```
generator client {
    provider        = "prisma-client-js"
    previewFeatures = ["views"]
}
```
My DBMS is MySQL.
I'm using Prisma/Client 4.11.
Thanks!

**Comment by janpio** [comment, marked as answer]
What code are you trying to execute that is giving you this error message?

**Reply by flaviovich** [reply]
Dear @janpio 
I've already got a solution by reading the documentation about [How to include views in your Prisma schema](https://www.prisma.io/docs/concepts/components/prisma-schema/views)
I'm following the [thread](https://github.com/prisma/prisma/issues/17335) about it.
Thank you for your reply.
